lib/puppet/type/user.rb

```diff
@@ -690,7 +690,9 @@ def delimiter
             :required_features => :libuser,
             :parent => Puppet::Parameter::Boolean) do
       desc "Forces the management of local accounts when accounts are also
-            being managed by some other NSS"
+            being managed by some other Name Service Switch (NSS).
+            
+            This option relies on your operating system's implementation of `luser...` commands, such as `luseradd` , and `lgroupadd`, `lusermod`. The `forcelocal` option could behave unpredictably in some circumstances. If the tools it depends on are not available, it might have no effect at all."
```

Comment:
> implementation of `luser...`

Is this how we've been addressing command families or methods that begin with the same thing? I feel like referred to them with a splat `*` at some point but (thinking I think of some set of hiera methods?) might be remembering wrong, or the style guide changed?

Reply:
"thinking I think" wow I need more coffee to english today 🤦‍♀

Reply:
I'm quite happy to change this to splat, I think I was just a little tired and accepted someone's suggestion tbh

Reply:
@jtappa , updated!